### PR TITLE
Revert c02a163

### PR DESCRIPTION
**Pull request by gabegorelick**
Not sure if https://github.com/garris/BackstopJS/commit/c02a1638f266d9e154a49541315af2f80312d265 was an intentional commit, but it has a lot of lint and syntax errors. The Docker image it generated is also broken:
```
Status: Downloaded newer image for backstopjs/backstopjs:latest
/usr/local/lib/node_modules/backstopjs/core/command/remote.js:13
    for await (const data of child.stdout) {
        ^^^^^

SyntaxError: Unexpected reserved word
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:617:28)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at requireCommand (/usr/local/lib/node_modules/backstopjs/core/command/index.js:44:26)
```  

See https://github.com/garris/BackstopJS/issues/861.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backstopjs",
-  "version": "3.6.0",
+  "version": "3.5.14",
```
Comment (gabegorelick) [inline package.json]: Will need to remove this from PR if these versions were already published.

**Comment by garris** [comment]
Not quite sure what this change does?

**Comment by garris** [comment]
Can’t you simply specify a tagged docker image?

**Comment by garris** [comment]
I will pull this because this probably should not have been pushed in the first place.

**Comment by gabegorelick** [comment]
@garris Thanks. I can confirm the `backstopjs/backstopjs:latest` image works now.

**Pull request merged**